server/export.go

```diff
@@ -109,7 +109,8 @@ func ExportCmd(appExporter types.AppExporter, defaultNodeHome string) *cobra.Com
 			return nil
 		},
 	}
-
+	cmd.SetOut(cmd.OutOrStdout())
+	cmd.SetErr(cmd.ErrOrStderr())
```

Comment:
shouldn't this be on all commands (i.e. on the root command)?

Reply:
Yes, it should. Unsure if set on the root command it would then propagate to the sub commands though.